pinot-controller/src/main/java/org/apache/pinot/controller/ControllerConf.java

```diff
@@ -1200,6 +1235,16 @@ private long convertPeriodToSeconds(String period) {
     return convertPeriodToUnit(period, TimeUnit.SECONDS);
   }
 
+  private boolean isValidPeriodWithLogging(String propertyKey, String periodStr) {
+    if (TimeUtils.isPeriodValid(periodStr)) {
+      return true;
+    } else {
+      LOGGER.error("Invalid time spec '{}' for config '{}'. Falling back to default config.", periodStr,
```

Comment:
Logging an error is not very user friendly because a lot of people don't really check logs. We should store these errors in the service starter, and hook them up with the UI so that user knows these configs are not picked up